tuplex/core/include/physical/PhysicalPlan.h

```diff
@@ -72,7 +72,8 @@ namespace tuplex {
         void executeWithParts(const tuplex::PhysicalPlan::File2FilePipeline &pip);
         double aggregateSamplingTime() const;
     public:
-
+        size_t getNumStages() const {return _num_stages;}
+        std::vector<LogicalOperator*> operators;
```

Comment:
this is a public field, can we change that into setter/getters? Also, shouldn't maybe a function `operators()` simply call `originalPlan->operators()` or so? I.e., the logical operators should be stored in the originalPlan (LogicalPlan) class.

Reply:
Done.